microhapdb/tables.py

```diff
@@ -35,6 +35,7 @@ def compile_variant_map(markers):
 merged = read_table("merged.csv")
 populations = read_table("population.csv")
 frequencies = read_table("frequency.csv.gz")
+frequencies["Count"] = frequencies["Count"].astype("Int16")
```

Comment:
Casting the `Count` column as a nullable integer type not only when writing to a file, but also when loading into memory.